.github/workflows/e2e_custom_cl.yml

```diff
@@ -223,7 +219,7 @@ jobs:
         with:
           should_checkout: true
           cl_repo: smartcontractkit/chainlink
-          cl_ref: ${{ env.CUSTOM_CORE_REF || github.event.inputs.cl_branch_ref }}
+          cl_ref: ${{ steps.get-core-ref.outputs.core-ref || github.event.inputs.cl_branch_ref }}
           dep_solana_sha: ${{ github.event.pull_request.head.sha }}
           push_tag: ${{ env.CL_ECR }}:solana.${{ github.sha }}
           QA_AWS_REGION: ${{ secrets.QA_AWS_REGION }}
```

Comment:
I can't comment further down but I think there are a couple other places that we use `Get core ref from PR body` below too

Reply:
Hmm... I see 2 more places in this file where it extracts "core ref" from the PR and writes it to `env.CUSTOM_CORE_REF`, but I'm not sure why those are there. It doesn't look like it's using it in any later step, since both of the jobs where that happens are testing already built chainlink images not checking out the source.

I also see it reading from `env.CUSTOM_CORE_REF` in `e2e_testnet_daily.yml`, but that doesn't make much sense either since it can't be triggered on a PR, only runs as a `workflow_dispatch` which already has an input `cl_branch_ref`.  I've removed that also.